### DISCUSSION TITLE
Can the open thread border router be ported to the Android platform?

### DISCUSSION THREAD

**Discussion by koenlee93** [openthread/openthread#9716] [Q&A] [answered]
I want to port open thread border router to Android platform.Is there any official tutorial like this?

**Comment by wgtdkp** [comment, marked as answer]
You can find useful links in https://github.com/orgs/openthread/discussions/9575

**Comment by koenlee93** [comment]
你好!我是李星宇，已收到你发的邮件，我会尽快处理的。
